<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(233, 223, 252);


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>190</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(233, 223, 252);


}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P_M">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_A">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		font: 75 11pt &quot;MS Shell Dlg 2&quot;;
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(214,     									196, 250, 255), stop:1 rgba(255, 255, 255, 255));
		border: 1px solid ; 
		border-color: rgb(150, 154, 225);
}
QPushButton:pressed {
		background-color: rgb(150, 154, 225);
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
